components/Dropdown/Dropdown.css

```diff
@@ -0,0 +1,17 @@
+.dropdown__button {
+    border: 1px solid black;
+    height: 40px;
+    margin-top:20px;
+    margin-bottom:20px;
+    margin-left:20px;
```

Comment:
Splitting hairs here, but you could try the margin shorthand syntax for this. It would be

``margin: 20px 0 20px 20px;``